src/EditorFeatures/Core/IntelliSense/AsyncCompletion/ItemManager.CompletionListUpdater.cs

```diff
@@ -140,12 +140,16 @@ private bool ShouldSelectSuggestionItemWhenNoItemMatchesFilterText
                     if (itemsToBeIncluded.Count == 0)
                         return HandleAllItemsFilteredOut();
 
+                    var highlightAndFilterTask = Task.Run(
```

Comment:
This is the only behavior change in this PR, we now calculate highlight and filter states in parallel to item selection.